resources/ts/Pages/Admin/User/Partial/UsersPartial.tsx

```
@@ -0,0 +1,91 @@
+                  checked={user.isAdmin}
+                  onChange={() => toggleAdmin(user.id, user.isAdmin)}
+                />
+                <div className="w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-blue-300 dark:peer-focus:ring-blue-800 rounded-full peer dark:bg-gray-700 peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all dark:border-gray-600 peer-checked:bg-blue-600"></div>
+                <span className="ml-3 text-sm font-medium text-gray-900 dark:text-gray-300">
+                  {user.isAdmin ? "Admin" : "Not admin"}
+                </span>
+              </label>
+            </li>
+          ))
+        )}
+      </ul>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)